.github/workflows/ci.yml

```diff
@@ -239,6 +240,47 @@ jobs:
             -e '{"awx_template_version": false}'
           testing-type: sanity
 
+      - name: Upload test coverage to Codecov
```

Comment:
Actually, `ansible-community/ansible-test-gh-action` above already performs uploads to Codecov internally.